code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -54,14 +54,23 @@
 	list_reagents = list("omnizine" = 30)
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF
 
+/obj/item/reagent_containers/hypospray/emergency_combat
+	name = "emergency combat stabilizer" //Used for the new combat medical kit.
+	desc = "A small modified air-needle auto injector, used by support operatives to quickly stabilize their allies in critical condition."
+	amount_per_transfer_from_this = 5
+	icon_state = "hypo"
+	volume = 35
+	ignore_flags = 1 // So they can heal their comrades.
+	list_reagents = list("oculine" = 5, "stimulants" = 15, "atropine" = 15)
+
 /obj/item/reagent_containers/hypospray/combat
 	name = "combat stimulant injector"
 	desc = "A modified air-needle autoinjector, used by support operatives to quickly heal injuries in combat."
 	amount_per_transfer_from_this = 10
 	icon_state = "combat_hypo"
 	volume = 90
 	ignore_flags = 1 // So they can heal their comrades.
-	list_reagents = list("epinephrine" = 30, "omnizine" = 30, "leporazine" = 15, "atropine" = 15)
+	list_reagents = list("tricordrazine" = 30, "omnizine" = 30)

```

Comment:
Their is a reason for it to give leporazine and epinephrine.

Reply:
leporazine and epinephrine makes no sense, when atropine is present. Your body does not lose heat, when it falls into a critical condition and epinephrine just locks your oxygen damage, which atropine already repeatedly heals on their own.

Reply:
And to clarify - the combat stimulant injector is not designed to stabilize you, its for healing purpose only. I made a seperate emergency combat stabilizer, just for that.

Reply:
Nukie hardsuits are not fire proof, people make flamers and such, lep was their to fix and stablize temp in case of Friendly fire with burn ammo or flamers. Epinephrine heals small amounts of stam as well as lock o2